homeassistant/components/huawei_lte/__init__.py

```diff
@@ -353,7 +362,11 @@ def signal_update() -> None:
         hass,
         NOTIFY_DOMAIN,
         DOMAIN,
-        {CONF_URL: url, CONF_RECIPIENT: config_entry.options.get(CONF_RECIPIENT)},
+        {
+            CONF_URL: url,
+            CONF_NAME: config_entry.options.get(CONF_NAME),
```

Comment:
Don't we need to set the default name here? Otherwise if the options are never activated by the user, they will be empty.

Reply:
That we do need to do indeed, fixed.